house_of_refuge/frontend/src/styles/resources.scss

```diff
@@ -452,17 +470,49 @@ a {
 
 .quick-filters {
   display: flex;
-  justify-content: space-between;
+  gap: 5px;
+  width: 100%;
   padding-bottom: 8px;
-  border-bottom: 1px solid $our-grey;
 }
 
 .quick-filter {
-  margin: 0 4px;
+  display: flex;
+  flex: 1 1 0;
+  flex-direction: column;
+
+  .react-dropdown-select {
+    width: 100%;
+  }
 
   label {
     margin-right: 6px;
   }
+
+  &__children {
+    height: 34px;
+    display: flex;
+
+    > div {
+      width: 100%;
```

Comment:
the switch buttons seem a bit heavy to me when they have full width, but maybe it's better to be consistent.. ideally I'd add some max-width to them

Reply:
Should be easier to use on mobile devices. I'll set a reasonable max width